### PR TITLE
perf(develop): don't use debug config for bluebird

### PR DESCRIPTION
**Pull request by pieh**
## Description

In `gatsby develop` we use debug build of `bluebird` currently which adds quite significant overhead. This is particularly visible on larger sites with large amount of nodes when we spawn tens of thousands `onCreateNode` runs.

 - In tested site `gatsby develop` was crashing with OOM error. This change allow to not hit this crash without adjusting `--max_old_space_size`.
 - Tested site with `--max_old_space_size=10000` before and after this change - source nodes step drops from ~67.87s (average across 3 runs) to ~21.7s (average across 3 runs)

## Related Issues

Fixes https://github.com/gatsbyjs/gatsby/issues/28916
Fixes https://github.com/datocms/gatsby-source-datocms/issues/123
Fixes https://github.com/datocms/gatsby-source-datocms/issues/134
[ch21729]

### CODE REVIEW

File: packages/gatsby/src/utils/api-runner-node.js [hunk 1]
```diff
@@ -33,6 +33,15 @@ const { trackBuildError, decorateEvent } = require(`gatsby-telemetry`)
 import errorParser from "./api-runner-error-parser"
 const { loadNodeContent } = require(`../db/nodes`)
 
+if (!process.env.BLUEBIRD_DEBUG && !process.env.BLUEBIRD_LONG_STACK_TRACES) {
+  // Unless specified - disable longStackTraces
+  // as this have severe perf penalty ( http://bluebirdjs.com/docs/api/promise.longstacktraces.html )
+  // This is main for `gatsby develop` due to NODE_ENV being set to development
```
Comment (KyleAMathews) [inline packages/gatsby/src/utils/api-runner-node.js:39]: s/main/mainly/

Comment (pieh) [reply packages/gatsby/src/utils/api-runner-node.js]: Addressed in https://github.com/gatsbyjs/gatsby/pull/28957/commits/92c5ced34ef041aea92267c8bf035a4e805f0dee

**Review by vladar — APPROVED** [review]
Great perf improvement! Thanks 👍

**Comment by vladar** [comment]
Published in `gatsby@2.30.2`

**Pull request merged**